src/apps/basicnat/basicnat.lua

```diff
@@ -0,0 +1,172 @@
+module(..., package.seeall)
+
+bit = require("bit")
+ffi = require("ffi")
```

Comment:
Better to localize variables when possible. I just point these ones, but there are more in the file I think.